src/txmempool.h

```diff
@@ -474,6 +474,12 @@ class CTxMemPool
     mutable uint64_t m_epoch;
     mutable bool m_has_epoch_guard;
 
+    // In-memory counter for external mempool tracking purposes.
+    // This number is incremented once every time a transaction
+    // is added or removed from the mempool(except when removed
+    // for block inclusion).
+    mutable uint32_t sequence_number{1};
```

Comment:
I believe my original theory was that the zmq sequence values themselves were also 32 bit, so I figured not much to be gained since any consumer has to handle this wrap around anyways?

Reply:
Offline someone noted that the ZMQ sequence numbers(probably) reset if the connection itself is reset, but the mempool sequence numbers will continue. I think this is a good argument to expand to `uint64_t`. Will do that soon.

Reply:
done